### DISCUSSION TITLE
Xây dựng phần comment đậm chất dev với github và giscus cho blog

### DISCUSSION THREAD

**Discussion by giscus** [nvtaikma/Comment-blogs#8] [General]
# Xây dựng phần comment đậm chất dev với github và giscus cho blog

http://localhost:5000/blog/xay-dung-phan-comment-dam-chat-dev-voi-github-va-giscus-cho-blog-1710659671963

**Comment by nvtaikma** [comment]
ádsad

**Reply by nvtaikma** [reply]
s

**Comment by nvtaikma** [comment]
> # Xây dựng phần comment đậm chất dev với github và giscus cho blog
> http://localhost:5000/blog/xay-dung-phan-comment-dam-chat-dev-voi-github-va-giscus-cho-blog-1710659671963
